extensions/ecc/tests/Cargo.toml

```diff
@@ -18,11 +18,13 @@ openvm-algebra-transpiler.workspace = true
 openvm-ecc-transpiler.workspace = true
 openvm-ecc-circuit.workspace = true
 openvm-ecc-guest.workspace = true
+openvm-pairing-guest = { workspace = true, features = ["bls12_381"] }
```

Comment:
it's a little weird that pairing is dependency in ecc, even if for test. can we move this test to `pairing/tests`?